# scene property
min_x: -1.0
max_x: 1.0
min_y: -1.0
max_y: 1.0
min_z: 1
max_z: 3
resolution: [480, 480]
fov: [1.57, 1.57]
rgbd_topic: rgbd
semantic_topic: semantic